### DISCUSSION TITLE
Left Join WHERE NULL

### DISCUSSION THREAD

**Discussion by madfatihid** [prisma/prisma#8302] [Q&A] [answered]
I'm trying to use Prisma to perform the following query to get the following result:
![Left Join WHERE NULL](https://www.postgresqltutorial.com/wp-content/uploads/2018/12/PostgreSQL-Join-Left-Join-with-Where.png)

```sql
SELECT
    *
FROM
    customers
LEFT JOIN
    orders
    ON customers.id = orders.customer_id
WHERE
    orders.customer_id IS NULL
```

How do I get this result with Prisma Client?

**Comment by ryands17** [comment, marked as answer]
@hmdfthid 👋 

This should work:

```ts
await prisma.customer.findMany({
  where: { order: { none: {} } }
})
```

You can read on how this is done [here](https://www.prisma.io/docs/support/help-articles/finding-entities-based-on-relation).

**Reply by madfatihid** [reply]
Wow, thank you @ryands17! I found the same link just as when you replied me. I wish there was a keyword of `LEFT OUTER JOIN` or something similar to it in the article so it's easier to find it by searching. But thanks anyways!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
